tests/test_cli.py

```diff
@@ -293,6 +299,48 @@ def test_correct_place_of_symlink_after_creation_of_it(self, mock_isfile):
              call(dst='/dst_folder/new_loc_add', src='/new_loc_add')],
             mock_symlink.call_args_list)
 
+    def test_delete_all_files_and_symlinks_in_destination_folder(self):
+        """ delete function should delete both file(s) and symlink(s) in destination folder. In this
+        test a temporary file and a symlink of it is created. Both of them should be deleted."""
+        with tempfile.TemporaryDirectory() as tmpdirname:
+            with tempfile.NamedTemporaryFile(dir=tmpdirname) as tmpfile:
+                os.symlink(src=tmpfile.name, dst=tmpfile.name + '_symlink')  # symlink creation
+                sys.argv = [
+                    "",
+                    '-b', "2038-06-01",
+                    '-e', "2038-06-09",
+                    '-d', tmpdirname,
+                    "--time_to_live", "0"
+                ]
+                with mock.patch('shutil.copy'):
+                    with mock.patch('os.remove') as mock_os_remove:
+                        cli_copy.main()
```

Comment:
Reduce your untitests to bare minimum:
```suggestion
                copier = Copier('type1', tmpdirname)
                copier.delete(0)
```

Reply:
done in the next commit